src/main/java/org/zalando/stups/tokens/AccessToken.java

```diff
@@ -37,26 +49,63 @@ protected AccessToken(final String token, final String type, final long initialV
         this.creationTimestamp = creationTimestamp;
     }
 
+    /**
+     * Get the token value of the {@link AccessToken}
+     *
+     * @return    The <i>token</i> {@link String} value of this {@link AccessToken}; will always be
+     * a non null value
+     */
     public String getToken() {
         return token;
     }
 
+    /**
+     * Get the type of the {@link AccessToken}
+     *
+     * @return  The <i>type</i> of this {@link AccessToken}, e.g. <i>Bearer</i>; will always be a
+     * non null value
+     */
     public String getType() {
         return type;
     }
 
+    /**
+     * The number of seconds this {@link AccessToken} was initially valid
+     *
+     * @return  The number of seconds this {@link AccessToken} was initially valid; will always be
+     * a positive <i>long</i> value
+     */
     public long getInitialValidSeconds() {
         return initialValidSeconds;
     }
 
+    /**
+     * The point in time until when this {@link AccessToken} is valud
+     *
+     * @return  A {@link Date} that will describe a point in time in the future in case this
+     * {@link AccessToken} is not yet expired (see {@link AccessToken#isExpired()}).
+     */
     public Date getValidUntil() {
         return validUntil;
     }
 
+    /**
+     * Tell whether this {@link AccessToken} is expired.
+     *
+     * @return  Will return {@link Boolean#TRUE} if this {@link AccessToken} is still valid, i.e.
```

Comment:
I'm not sure about this. If the AccessToken is expired I would expect an TRUE as return value. If the AccessToken is still valid, I would expect an FALSE as return value, right?